core/src/test/java/org/apache/accumulo/core/spi/scan/ConfigurableScanServerSelectorTest.java

```diff
@@ -490,7 +490,7 @@ public void testWaitForScanServers() {
     // will wait for scan servers
 
     String defaultProfile =
-        "{'isDefault':true,'maxBusyTimeout':'5m','busyTimeoutMultiplier':4,'enableTabletServerFallback':false,"
+        "{'isDefault':true,'maxBusyTimeout':'5m','busyTimeoutMultiplier':4,'timeToWaitForScanServers':120ms,"
```

Comment:
What value is used to disable this? Is it `0` ? I'm thinking that this should be documented (how to disable). Also, I think the default value should be whatever it needs to be to disable this as I think that keeps the behavior the same.

Reply:
The values do not have to numeric because we parse them.  We could accept a timeout value of `never`, if its documented, and in the impl set that to max long.

Reply:
Would be good to make `0` value disable it and document that.

Reply:
I agree with having `0` be the disable value. I think with setting duration keeping things consistent with numerical values is best. I'll add a note in the docs about `0` being the default and disable value.